cmd/kogito/command/project/use_project.go

```diff
@@ -96,9 +97,9 @@ func (i *useProjectCommand) Exec(cmd *cobra.Command, args []string) error {
 
 		log.Infof(message.ProjectSet, i.flags.project)
 
-		install := shared.ServicesInstallationBuilder(i.Client, ns.Name).SilentlyInstallOperator()
+		install := shared.ServicesInstallationBuilder(i.Client, ns.Name).SilentlyInstallOperatorIfNotExists()
```

Comment:
Same here if Infinispan/Strimzi CRDs are not there

Reply:
See my other comment.